### PR TITLE
Add amt+null backend to installed files

### PR DESCRIPTION
**Pull request by okurz**
* Install the "null" backend as well
* Add "amt" backend and console to installed files

### CODE REVIEW

File: tools/check-installed-files [hunk 1]
```diff
@@ -3,7 +3,7 @@ destdir="${destdir:-$(mktemp -d)}"
 make install DESTDIR="$destdir"
 installed_files=$(find "$destdir" -name '*.pm')
 all_source_files=$(git ls-files "*.pm" 2>/dev/null || find "$(dirname "$0")" -name "*.pm" -printf "%P\n")
-source_files=$(echo "$all_source_files" | grep -v -E '(ppmclibs/|t/|tools/|backend/amt|backend/null|consoles/amtSol)')
+source_files=$(echo "$all_source_files" | grep -v -E '(ppmclibs/|t/|tools/)')
```
Comment (kalikiana) [inline tools/check-installed-files:6]: So this even *tests* that we don't install these backends? :thinking:

Comment (Martchus) [reply tools/check-installed-files:6]: He's only added this test recently. (See his message in Rocket Chat.)

Comment (okurz) [reply tools/check-installed-files:6]: Yes, it tested that all files except this "blacklist" were installed as a consequence of the last major "all openQA instances product only incompletes" havoc that you might have missed :)

Comment (kalikiana) [reply tools/check-installed-files:6]: Sorry if I'm getting confused by the details, it just looks a bit odd that we intentionally don't install it, backed up by tests, only to ask why we don't install it ;-)

**Pull request merged**